### DISCUSSION TITLE
Struggling with Student Developer Pack Application - Getting Rejected, Need Help!

### DISCUSSION THREAD

**Discussion by pravo123** [community/community#139534] [GitHub Education]
### Hi everyone,

Hi everyone,

I'm facing persistent issues with my GitHub Student Developer Pack application and could really use some advice. Here's my situation:

_**Application History**_
- Applied twice in the last month
- Keep getting rejections

**_Main Issues_**

**Verification Process:**
The only option is to take a live picture:
- Webcam pictures are blurry
- iPhone back camera isn't functioning for this
- Can't capture all details clearly in one shot
No option to upload documents directly

**Student Status Verification:**
- I'm enrolled in DeVry Keller Graduate School of Management
- Taking courses online, so I'm not in the college vicinity
- Don't have a physical student ID (not required by my school)
- Have a valid transcript, enrollment cert but can't upload it due to system limitations

**Rejection Reasons:**
- Initially: "Not in college area", 2FA, and profile issues
- My 2FA and profile was updated for the second try. But it still showed me this. 

**_What I've Tried_**
- Attempted various verification methods offered within the system
- Looked for GitHub support options, but only found directions to post in the forum

_**What I Need**_
1. Alternative way to verify my student status
2. Advice on how to proceed given my online student status
3. Tips for getting a clear, compliant photo of my transcript using the live picture option
4. Method to upload the official transcript and enrollment cert. would be sweet!

I'm really keen on using GitHub Copilot for my coding journey. Any help or advice would be greatly appreciated!
Thank you in advance for your assistance.

**Comment by stefisha** [comment]
Hey there! Sorry to hear you're having issues with the GitHub Student Developer Pack. Here's some advice that might help:

1. For the live picture issue:
- I know the system asks for a live picture, but it needs to be clear enough for them to read. If your webcam and iPhone back camera aren’t working well, you could try using the front camera of your phone or a different device, like a friend's phone or laptop.
- Make sure the lighting is good and the document is flat. If your picture comes out blurry, retake it a few times until it’s readable. It’s annoying, but a clear shot is important.

2. Alternative proof since you don't have a physical student ID:
- If you have your transcript or an enrollment certificate, even though you can’t directly upload them, you can still take a photo of them using the live picture option. Just be sure they show your name, school, and a current date. You can also combine multiple documents in one image to cover all the requirements.

3. Online student status:
- If the system is flagging you because you’re not in the "college area," that’s probably because you're enrolled online. GitHub still accepts proof from online schools, so make sure your documents clearly show you’re enrolled in the school.
- Also, try mentioning in the description when you apply that you're taking courses online and don’t have a physical ID since it’s not required by your school. That might help them understand your situation better.

More options:
If you don’t have an academic email, check out this FAQ: [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details). It explains what to do when you’re stuck with email verification issues.

Final tips:
Since you updated your 2FA and profile, that should help, but keep an eye on common rejection reasons. Sometimes it’s a minor detail that gets overlooked. Try this link for more info on rejections.
I hope this helps, and good luck getting it sorted! Let me know if you need anything else!

**Reply by pravo123** [reply]
How do I take pictures of multiple documents? I can only take 1.
